db/schema.rb

```diff
@@ -249,6 +250,9 @@
     t.index ["uuid"], name: "index_courses_on_uuid", unique: true
   end
 
+  create_table "data_migrations", primary_key: "version", id: :string, force: :cascade do |t|
```

Comment:
I think we decided to leave this out ages ago?

Reply:
think another rebase artefact   - I've corrected the schema now